### PR TITLE
Add device delete and edit operation

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/device.js [hunk 1]
```diff
@@ -33,13 +33,11 @@ deviceModule = function () {
...
     var CommandOperation = Packages.org.wso2.carbon.device.mgt.core.operation.mgt.CommandOperation;
 
-    var deviceManagementService = utility.getDeviceManagementService();
-
-    var devicemgtService;
-
     var publicMethods = {};
     var privateMethods = {};
 
+    var deviceCloudService = devicemgtProps["httpsURL"] + "/common/device_manager";
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/device.js:39]: You camelcase correctly.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ policyModule = function () {
             var queName = "wso2/iot/" + carbonUser.username + "/" + deviceType;
             log.info("Queue : " + queName);
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:43]: Does this log necessary?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/device.js (hunk 1 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/device.js:39]: There is no any variable which use incorrect case.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js (hunk 2 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:43]: Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

**Pull request merged**